docs/requirements.txt

```diff
@@ -0,0 +1 @@
+m2r2==0.2.7
```

Comment:
Should be added to poetry development requirements.

Reply:
I added this to poetry. I believe this also has to be in the requirements file for readthedocs. That's my understanding at least. I can verify that.